docs/design/import.md

```diff
@@ -0,0 +1,314 @@
+# Import
+
+Practitioners may have previously existing resources that they wish to bring under Terraform'm management. Conceptually, this is called importing as Terraform must create the state of the resource. Providers are responsible for receiving a request for resource import and respond with the relevant resource state(s).
+
+This design documentation will walk through and recommend options for import handling in the framework.
+
+## Background
+
+### Terraform CLI
+
+Terraform CLI supports two forms of interacting with unmanaged resources:
+
+- [`terraform add`](https://www.terraform.io/docs/cli/commands/add.html) (*Experimental*, Terraform CLI version 1.1.0 and later): Given a resource address and using the resource schema provided by the provider, generate a configuration template.
```

Comment:
This isn't live yet, so could switch it to something like https://github.com/hashicorp/terraform/blob/43d753e727c4248e8d5bf8a703b1937f75c55c76/website/docs/cli/commands/add.html.md, if desired.

Reply:
I think leaving it as it is is probably fine, I don't think it will matter soon (for some value of "soon", I have no idea when 1.1 is planned for).